docker/lr-align/Dockerfile

```diff
@@ -23,10 +23,10 @@ COPY merge_read_group_tags.py /usr/local/bin/
 COPY cat_as_fastq.py /usr/local/bin/
 
 # install minimap2
-RUN wget https://github.com/lh3/minimap2/releases/download/v2.17/minimap2-2.17_x64-linux.tar.bz2 \
-        && tar xjf minimap2-2.17_x64-linux.tar.bz2 \
-        && rm minimap2-2.17_x64-linux.tar.bz2 \
-        && cd minimap2-2.17_x64-linux \
+RUN wget https://github.com/lh3/minimap2/releases/download/v2.24/minimap2-2.24_x64-linux.tar.bz2 \
```

Comment:
minor: define an `ARG` for these version info? easier to maintain.

Reply:
👍